wire-tests/src/test/java/com/squareup/wire/java/KotlinGeneratorTest.kt

```diff
@@ -0,0 +1,97 @@
+package com.squareup.wire.java
+
+import com.squareup.kotlinpoet.FileSpec
+import com.squareup.wire.kotlin.KotlinGenerator
+import com.squareup.wire.schema.RepoBuilder
+import junit.framework.TestCase.assertTrue
+import org.junit.Test
+
+class KotlinGeneratorTest {
+  @Test fun basic() {
+    val repoBuilder = RepoBuilder()
+      .add("message.proto", """
+        |message Person {
+        |	required string name = 1;
+        |	required int32 id = 2;
+        |	optional string email = 3;
+        |	enum PhoneType {
+        |		HOME = 0;
+        |		WORK = 1;
+        |		MOBILE = 2;
+        |	}
+        |	message PhoneNumber {
+        |		required string number = 1;
+        |		optional PhoneType type = 2 [default = HOME];
+        |	}
+        |	repeated PhoneNumber phone = 4;
+        |}""".trimMargin())
+    val code = repoBuilder.generateKotlin("Person")
+    assertTrue(code.contains("data class Person @JvmOverloads constructor"))
```

Comment:
Can we do equality testing? It'll be easier to understand what the output looks like

Reply:
The entire code is going to be pretty big, that's why I was avoiding it